dynatrace/src/test/java/com/dynatrace/opentelemetry/metric/SerializerTest.java

```diff
@@ -233,7 +233,6 @@ void createInvalidDoubleGaugeLines() {
     Collection<DoublePointData> doublePointDataCollection =
         new ArrayList<DoublePointData>() {
           {
-            add(DoublePointData.create(0L, 0L, Labels.empty(), 123.456d));
```

Comment:
You could move this to `createDoubleGaugeLines` (with a different value so it's distinguishable) and assert on it without the timestamp.